runtime/moonbase/tests/integration_test.rs

```diff
@@ -2732,7 +2732,7 @@ fn test_xcm_utils_weight_message() {
 
 		Precompiles::new()
 			.prepare_test(ALICE, xcm_utils_precompile_address, input)
-			.expect_cost(1669)
+			.expect_cost(2368)
```

Comment:
```suggestion
			.expect_cost(<Runtime as frame_system>::DbWeight::get().read.saturating_div(WEIGHT_PER_GAS))
```

Reply:
Nice! Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3459/commits/0dc8157f9969b0136f7a2f653e9946dbd9b02e5d